### PR TITLE
build: add imagetools examples for inspecting attestations

### PR DESCRIPTION
**Pull request by jedevc**
:arrow_up: Follow-up to https://github.com/docker/buildx/pull/1498#pullrequestreview-1241070105.

### CODE REVIEW

File: build/attestations/sbom.md [hunk 1]
```diff
@@ -168,6 +168,42 @@ sbom-hugo.spdx.json
...
+    --format "{{ json .SBOM.SPDX }}"
+{
+  "SPDXID": "SPDXRef-DOCUMENT",
+  ...
+}
+```
+{% endraw %}
+
+You can also construct more complex expressions using the full functionality
+of go templates. For example, you can list all the installed packages and their
```
Comment (dvdksn) [inline build/attestations/sbom.md:192]:
```suggestion
of Go templates. For example, you can list all the installed packages and their
```

File: build/attestations/slsa-provenance.md [hunk 2]
```diff
@@ -142,7 +142,41 @@ using build arguments, consider refactoring builds to pass secret values using
...
+    --format "{{ json .Provenance.SLSA }}"
+{
+  "buildType": "https://mobyproject.org/buildkit@v1",
+  ...
+}
+```
+{% endraw %}
+
+You can also construct more complex expressions using the full functionality of
+go templates. For example, for provenance generated with `mode=max`, you can
```
Comment (dvdksn) [inline build/attestations/slsa-provenance.md:166]:
```suggestion
Go templates. For example, for provenance generated with `mode=max`, you can
```

**Review by dvdksn — COMMENTED** [review]
nice to get this in!

**Review by dvdksn — APPROVED** [review]
:shipit:

**Comment by jedevc** [comment]
@dvdksn can you merge :tada: I don't have permissions :eyes:

**Pull request merged**